docs/debugging-python-lambda-functions.md

```diff
@@ -0,0 +1,92 @@
+# Debugging Python lambda functions
+
+## Install and configure prerequisites
+
+1. Install [Docker](https://www.docker.com/get-started) and start the Docker service.
+2. Install [AWS SAM CLI](https://github.com/awslabs/aws-sam-cli/releases).
```

Comment:
Make it clear that the user also needs to install the AWS CLI. Possibly link here instead: https://aws.amazon.com/serverless/sam/ (this spells out those requirements and has links to executables)